src/main/java/models/Game.java

```diff
@@ -7,25 +7,27 @@
 /**
  * Created by michaelhilton on 1/25/16.
  */
-public class Game {
+public abstract class Game {
 
     public java.util.List<Card> deck = new ArrayList<>();
 
     public java.util.List<java.util.List<Card>> cols = new ArrayList<>();
 
-    public int score;
+    public int score = 0;
 
-    public int cardsLeft;
+    public int cardsLeft = 0;
 
     public Game(){
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
+        this.buildDeck();
```

Comment:
the shuffle() is just below the buildDecl method

Also the shuffle method is called after initializing the game in the Application Controller